kloppy/tests/test_opta.py

```diff
@@ -28,6 +28,7 @@ def test_correct_deserialization(self):
         assert dataset.metadata.provider == Provider.OPTA
         assert len(dataset.events) == 17
         assert len(dataset.metadata.periods) == 2
+        assert dataset.events[15].ball_owning_team == dataset.metadata.teams[0]
```

Comment:
Can you update this a little bit so we know:
1. It works for events that are in the list of `BALL_OWNING_EVENTS`
2. It works for events that are not in the list of `BALL_OWNING_EVENTS`